pkgs/applications/audio/virtual-ans/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv
+, fetchurl
+, unzip
+, xorg
+, libGL
+, alsaLib
+, SDL2
+, autoPatchelfHook
+}:
+
+stdenv.mkDerivation rec {
+  name = "virtual-ans-${version}";
+  version = "3.0.2c";
+
+  src = fetchurl {
+    url = "https://warmplace.ru/soft/ans/virtual_ans-${version}.zip";
+    sha256 = "1dd4d91yjmp7xfjcjlam8l55r4963ypjgll97n4gjvl72m1fq856";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Photoelectronic microtonal/spectral musical instrument";
+    homepage = "https://warmplace.ru/soft/ans/";
+    license = licenses.free;
+    platforms = platforms.linux;
+    maintainers = with maintainers; [ jacg ];
+  };
+
+  nativeBuildInputs = [
+    unzip
+    autoPatchelfHook
+    stdenv.cc.cc.lib
+    xorg.libX11
```

Comment:
You should't be using `xorg.*` here, rather import the libraries you need directly. 
```
{ stdenv
...
, libXi
, libX11
...
}:

...

buildInputs = [
  libX11
  libXi
  ...
];
```
(Apologies for the weird formatting :sweat_smile:)